### ISSUE TITLE
Disable User Input/Travelling Deviations After Certain Point On Road

### ISSUE THREAD

**Issue by anthonyromyn** [DuncanLab/RoadRAGE#10]
Hi RoadRAGE Team,

One thing that would be super helpful/necessary for our studies would be the ability to turn off user input after a certain point on a road/map. I am thinking here of two scenarios:

1. With the default .json and prefab maps, if a user presses an arrow key left or right after the fork in the road has already been passed, the car will turn around and try to return to node #1. It would be great to have a point on the map, after which the movement keys no longer work.

2. I've noticed while testing the software that if I press an arrow key many times before the fork, the car will travel through the fork correctly but then sometimes turn around and head back to node #1 later on, based on the presses I made before the fork. So, if the no-more-input point requested in #1 above could also remove this, that would be really helpful. Unfortunately participants will definitely press the arrow buttons a bunch of times, even if we tell them not to. 

In other words, to summarize both point 1 and 2 in a succinct manner, it would be really helpful to have an option in the software where we can specify that after x point on the road/map, the car will just continue travelling down that road no matter what buttons were pressed previously or what buttons will be pressed in the future.

Thanks!
Anthony

**Comment by codyhowarth** [comment]
So both problems _should_ be fixed with my changes for #7.

However, the ability to shut off user input at a certain time is still a valid feature request, so I'm going to spin that off into it's own issue.
